src/utils/rtkUtil.ts

```diff
@@ -59,6 +60,14 @@ export const apiIdentityWalletQuery = () => ({
   },
 })
 
+export const apiSSICredentialQuery = () => ({
```

Comment:
@manojava-gk I already created the function for SsiCredential on line 39. Please use that one. If not using anywhere then can you please remove that one.

Reply:
removed. Thanks